### PR TITLE
passed quotation data to front end for being recorded by FullStory

### PR DESCRIPTION
**Pull request by sushantnadkar**

### CODE REVIEW

File: awesome_cart/power.py [hunk 1]
```diff
@@ -40,6 +41,10 @@ def get_power_user_settings():
 
 	user_doc = frappe.get_doc("User", frappe.session.user)
 
+	# pass quotation with response object to be recorded by FullStory
+	session_quotation = get_user_quotation(get_awc_session())
```
Comment (vjFaLk) [inline awesome_cart/power.py]: Does this get the entire quotation or just the name?

File: awesome_cart/public/js/client/awc.erpnext.adapter.js [hunk 2]
```diff
@@ -1002,6 +1002,8 @@ $(function () {
 					});
 				}
 			}
+			// trigger FullStory to record custom data
+			$("body").trigger("awc-power-user-settings", data);
```
Comment (vjFaLk) [inline awesome_cart/public/js/client/awc.erpnext.adapter.js]: When does this get triggered? Are we sure it's triggered always? To make sure that Fullstory is tracking everything?

File: awesome_cart/power.py (hunk 1 above)
Comment (sushantnadkar) [reply awesome_cart/power.py]: entire quotation, should i just send the 2 fields i we need?

File: awesome_cart/public/js/client/awc.erpnext.adapter.js (hunk 2 above)
Comment (sushantnadkar) [reply awesome_cart/public/js/client/awc.erpnext.adapter.js]: yeah this code is executed on all the pages

**Comment by vjFaLk** [comment]
https://github.com/DigiThinkIT/awesome_cart/pull/285

Merged https://github.com/DigiThinkIT/awesome_cart/pull/285 as a better alternative

**Pull request closed without merging**